test/runner/tob.ts

```diff
@@ -120,7 +120,7 @@ function outputBlock(tob: Tob, updateItems: any) {
   return `
 ## Output
 
-### Hybrid CST
+### CST
```

Comment:
why changing this?

Reply:
It's not an hybrid CST.

Reply:
you have to change the read logic - if want to do this.

Reply:
can you fix that and merge?

Reply:
and you also need to update all the md files. or the tests will be failing.

I would say, better to do it in another PR - one PR for one thing.

Reply:
okay. I'm reverting it and let you look into it.